its/src/test/java/com/sonar/maven/it/suite/JavaTest.java

```diff
@@ -120,6 +121,31 @@ void setJavaVersionProperties() throws IOException {
       entry("sonar.java.target", "1.8"));
   }
 
+  @Test
+  void whenPropertyJavaHomeExistsThenItIsUsedAsDefaultValue() throws IOException {
+    //verify that property java home exists, it may not be pointing to a jdk
+    if (!System.getProperties().containsKey("java.home")
+      && !System.getenv().containsKey("JAVA_HOME")) {
+      return;
+    }
```

Comment:
Is there a chance the value does not exist? If it is possible, then we should use a more idiomatic of verifying that we can test using [assumeThat for instance](https://github.com/assertj/assertj-examples/blob/0868b5d724374ca0eb3f6c2456b27acd5ac740e0/assertions-examples/src/test/java/org/assertj/examples/AssumptionExamples.java#L41).

Reply:
:+1: